pkg/controllers/suite_test.go

```diff
@@ -37,7 +37,7 @@ import (
 // These tests use Ginkgo (BDD-style Go testing framework). Refer to
 // http://onsi.github.io/ginkgo/ to learn more about Ginkgo.
 
-var cfg *rest.Config
+var _ *rest.Config
```

Comment:
is this needed?

Reply:
Probably when we start using `gingko` based BDD.